micrometer-core/src/main/java/io/micrometer/core/instrument/MeterRegistry.java

```diff
@@ -300,6 +300,8 @@ Meter register(Meter.Id id, Meter.Type type, Iterable<Measurement> measurements)
      * @return The set of registered meters.
      */
     public List<Meter> getMeters() {
+        // NOTE: previously used pcollections HashTreePMap which consistently ordered by the key's
```

Comment:
Maybe this is better on release notes instead of code

Reply:
agreed